### ISSUE TITLE
Just asking a questions.

### ISSUE THREAD

**Issue by huajideshutiao** [Xposed-Modules-Repo/com.amanoteam.sunny#1]
What does rate limiting in the readme refer to specifically? Is it a limit when updating?

**Comment by Kartatz** [comment]
The "rate limit" mentioned in the README refers to a feature that several Tachiyomi extension maintainers use to limit the maximum number of requests that can be made in a given amount of time.

This limitation is most used to prevent manga sources from being overloaded (see https://github.com/tachiyomiorg/tachiyomi-extensions/pull/17032). In some cases, this limitation is added at the request of the site owner (see https://github.com/tachiyomiorg/tachiyomi-extensions/issues/5919).

In short, this feature intentionally slows down your downloads, browsing, and manga reading.

"But how much slow?":

![sunny.mp4](https://raw.githubusercontent.com/Kartatz/r/main/sunny.mp4)

Module enabled / Module disabled
